### DISCUSSION TITLE
Settings dimming regressed UX

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#14414] [Q&A] [answered]
Previously, whenever an option was in view you could toggle/change it. In the current release though, it is very likely that the section you are clicking on is dimmed, scrolling your view instead. That forces you to move cursor back to the option and click again to change it. I found the dimmed version very frustrating to use the way it is now
Here's a visual comparison between the previous release and the latest:

https://user-images.githubusercontent.com/53346335/130277445-e017b4be-0517-45fb-b9ff-cb7fefca13ff.mp4

**Comment by frenzibyte** [comment]
This is a known issue with the sections container implementation.

**Comment by peppy** [comment, marked as answer]
This is mostly intended. We may tweak the vertical point as which a section becomes current, but please give it a bit of time and see if you get used to it.

**Comment by KoleckOLP** [comment]
I think it's vastly worse experience because I am tying to press something on the grayed out portion of the screen, then the setting jump around and I lost track of what I wanted to press.

It also makes looking for a setting worse, I scroll the setting and more than not the settings are grayed out so just skimming the settings is worse.

What was wrong with the original implementation?
